cmd/command/recovery.go

```diff
@@ -85,22 +89,46 @@ func recoverObjectAction(ctx *cli.Context) error {
 	}
 
 	if ctx.IsSet(objectListFlag.Name) {
-		if ctx.NArg() < 1 {
-			return fmt.Errorf("args number error")
-		}
-		argNum := ctx.NArg()
-		for i := 0; i < argNum-1; i++ {
-			recoverObject(bucketName, ctx.Args().Get(i), cfg, client)
+		objectNames := ctx.String(objectListFlag.Name)
+		objectNameList := strings.Split(objectNames, "//")
+		for _, objectName := range objectNameList {
```

Comment:
if len(objectNameList) == 0, return err

Reply:
If there is nothing after objectListFlag -l, the system will throw an error "flag needs an argument: -l". If there is anything after objectListFlag -l, objectNames will not be an empty string, and objectNameList := strings.Split(objectNames, "//") will never result in  len(bjectNameList) ==0 (e.g., strings.Split("example", "//") will result in [example]. Thus there is no need to add if len(objectNameList) == 0 check.